src/main/java/pl/sda/carrentalmanager/services/RentalService.java

```diff
@@ -0,0 +1,20 @@
+package pl.sda.carrentalmanager.services;
+
+import org.springframework.stereotype.Service;
+import pl.sda.carrentalmanager.models.Rental;
+import pl.sda.carrentalmanager.repositories.RentalRepository;
+
+import java.util.List;
+
+@Service
+public class RentalService {
+    private RentalRepository rentalRepository;
+
+    public RentalService(RentalRepository rentalRepository) {
+        this.rentalRepository = rentalRepository;
+    }
+
+    public List<Rental> getAll() {
```

Comment:
Dobrą praktyką jest trzymanie jednakowych nazw, czyli jeśli w repository jest metoda findAll() to w serwisie tez powinna sie tak nazywac